views/movies/edit-movie.hbs

```diff
@@ -0,0 +1,37 @@
+<h2>Edit Movie</h2>
+
+<form action="/movies/{{foundMovie._id}}/edit" method="POST">
+
+  <p>   
+        <label>Title:</label>
+        <input type="text" name="title" />
+    </p>
+
+    <p>
+        <label>Genre:</label>
+        <input type="text" name="genre" /> 
+    </p>
+
+  <p>
+        <label>Plot:</label>
+        <input type="text" name="plot" />
+  </p>
```

Comment:
It's a good practice to use a placeholder in the input fields, so that the user can see what they're going to change

```suggestion
<p>   
        <label>Title:</label>
        <input type="text" name="title" placeholder="{{foundMovie.title}}"/>
    </p>

    <p>
        <label>Genre:</label>
        <input type="text" name="genre" placeholder="{{foundMovie.genre}}"/> 
    </p>

  <p>
        <label>Plot:</label>
        <input type="text" name="plot" placeholder="{{foundMovie.plot}}" />
  </p>
```